### DISCUSSION TITLE
What language is "Hakuna Matata" from? 🦁

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1216] [Galaxy Trivia] [answered]
**Bonus:** According to legend, at whom did Peeping Tom peep? 🫣

**Comment by peterbaccaro** [comment, marked as answer]
"Hakuna matata" is actually a phrase in the East African language of **Swahili** that literally means “no trouble” or “no problems”.

According to legend, Peeping Tom peeped at **Lady Godiva** and was instantly struck blind.
